requirements.txt

```diff
@@ -1,13 +1,38 @@
 anybase32==1.1.0
+blessed==1.19.1
 boto3==1.17.93
 botocore==1.20.93
+certifi==2021.10.8
+cffi==1.15.0
 cfn-flip==1.2.3
+chardet==4.0.0
 click==8.0.1
 deepdiff==5.5.0
+Deprecated==1.2.13
```

Comment:
did you run the `pip freeze` in a separated env? meaning, are all this dependencies necessary or you included you system dependencies without realizing?

Reply:
Let me check again. I don't think I included all the system dependencies. I ran `pip freeze` after activating the environment.

Reply:
Yes, these are the requirements that appear after running `pip freeze` from inside the environment. Pretty sure that it doesn't include packages that appear outside the environment. This long list includes dependencies of installed packages as well.

Some dependencies of the packages that we are using (e.g. numpy for pandas) is not included in the  previous requirements.txt.

There is this package that would let us only specify the "head" packages into requirements without inserting the dependencies: https://pypi.org/project/pip-chill/